src/js/media-container.js

```diff
@@ -232,11 +232,11 @@ class MediaContainer extends window.HTMLElement {
         new window.CustomEvent(eventName, { composed: true, bubbles: true })
       );
     };
+    media.addEventListener('click', this._mediaClickPlayToggle, false);
   }
 
   handleMediaUpdated(media) {
     const resolveMediaPromise = (media) => {
-      // media.addEventListener('click', this._mediaClickPlayToggle, false);
```

Comment:
`mediaSetCallback` is called after `handleMediaUpdated`, so, `this._mediaClickPlayToggle` was never set. Instead, we can move this listener into `mediaSetCallback` to match it being removed from `mediaUnsetCallback`